test/src/edu/stanford/nlp/util/AcronymMatcherTest.java

```diff
@@ -19,6 +20,14 @@ public void testBasic() {
     assertTrue(AcronymMatcher.isAcronym("ECC", "Election Complaints Commission".split("\\s+")));
   }
 
+  @Test
```

Comment:
Mind adding some comments for what this is testing?

Reply:
A is a stopword, which is why the commented out line isn't working, right?

Reply:
yes, sorry my bad, will remove it!

Reply:
That's fine.  In fact I just left that test commented out with a note of why it wasn't working

Reply:
Rebasing off the current master would make it easier to make more progress on this change.  Thanks for the contribution!